packages/cbioportal-frontend-commons/src/components/WindowWrapper.ts

```diff
@@ -32,4 +32,14 @@ export default class WindowWrapper {
             height: this.windowObj.innerHeight,
         };
     }
+
+    @autobind
+    public getWindowWidth() {
```

Comment:
maybe make these computed getters since they reference observables?

Reply:
i say this only so that it's clear to callers that they are getting observables

Reply:
I specifically made them of this form for uses like this: https://github.com/cBioPortal/cbioportal-frontend/pull/3152/files#diff-b02249405d34f258fb2e74c6a0e83c73R845

users can use .size.width if they want a straight value. and I think if someone is already looking at this file then @computed doesnt add anything that the @observables above don't already